api/authenticationHelpers.ts

```
@@ -1,12 +1,13 @@
+import { Request, Response, NextFunction } from 'express';
 
-function authenticateUser(req, res, next) {  
+function authenticateUser(req: Request, res: Response, next: NextFunction): void {  
     if (!req.user) {  
         return res.status(401).json({ message: 'Authentication required' });  
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)